### ISSUE TITLE
GPU resource for re-produce results

### ISSUE THREAD

**Issue by hoanganhpham1006** [salesforce/BiST#3]
Hi,

I'm trying to reproduce your results on AVSD@DSTC7 dataset. I extracted the resnext features and run the exec.sh bash script with all default setting (s2t and t2s both are set to 1). I used the V100 GPU with 32GB VRAM, however, I only can set the batchsize to 1 otherwise is OOM. 

Should I change anything in the configure to reproduce the results reported in paper?

**Comment by hoanganhpham1006** [comment]
@henryhungle Hope to get your response

**Comment by henryhungle** [comment]
Hi @hoanganhpham1006 have you tried to use multiple GPU resources? e.g. CUDA_VISIBLE_DEVICES=0,1,2..

**Comment by hoanganhpham1006** [comment]
When I tried multiple GPU settings, it threw another error in the DDP model. 

It seems like there is a leak memory issue that I'm trying to find out since the memory is accumulated every training step then OOM.

**Comment by henryhungle** [comment]
@hoanganhpham1006 I do not recall having the leak memory issue during my experiments. Let me try again with the current version of the repo.
